src/export.cpp

```diff
@@ -92,8 +91,27 @@ void SolveSpaceUI::ExportSectionTo(const Platform::Path &filename) {
         sb->auxA = Style::SOLID_EDGE;
     }
 
-    el.CullExtraneousEdges();
-    bl.CullIdenticalBeziers();
+    el.CullExtraneousEdges(true);
```

Comment:
Would be good to have `/*both=*/` here and elsewhere because it looks quite puzzling otherwise.